docroot/themes/custom/mass_theme/templates/block/block--mainnavigation-2.html.twig

```diff
@@ -35,13 +35,14 @@
 Desktop horizontal main nav is added by themes/custom/mass_theme/templates/block/block--mainnavigation-2.html.twig
 in the following block to this.
 #}
-{% set ariaLabel = heading_id ?? 'main-navigation-heading' %}
-<nav class="ma__header__hamburger__nav" aria-labelledby="{{ ariaLabel }}" id="main-navigation" role="navigation">
+{# {% set ariaLabel = heading_id ?? 'main-navigation-heading' %} #}
+<nav class="ma__header__hamburger__nav" id="main-navigation" role="navigation">
```

Comment:
```suggestion
<nav class="ma__header__hamburger__nav" id="main-navigation" aria-label="main navigation">
```

Please see my comment in the mayflower PR for this https://github.com/massgov/mayflower/pull/1743